c/tskit/tables.c

```diff
@@ -9746,6 +9876,13 @@ tsk_table_collection_subset(tsk_table_collection_t *self, const tsk_id_t *nodes,
         }
     }
 
+    /* Rewrite the individual parent ids */
+    for (k = 0; k < (tsk_id_t) self->individuals.parents_length; k++) {
+        if (self->individuals.parents[k] != TSK_NULL) {
+            self->individuals.parents[k] = individual_map[self->individuals.parents[k]];
```

Comment:
We should try to cover this in tests? (or is that coming in one of the follow-ups?)

Reply:
yes, this is in #1226.